README.md

```diff
@@ -201,6 +212,14 @@ The following documentation is available:
 <a name="known-issues"></a>
 ## Known Issues
 
+<a name="bugs"></a>
+### Bugs
+
+For a list of current bugs, see the [release notes](https://github.com/minishift/minishift/releases).
```

Comment:
Aha ok, so it only lists fixed bugs. We need to find a way to mention currently open bugs (which translate to known issues), but that's tracked in a different issue. For now I'll remove this.